src/fluree/db/query/turtle/parse.cljc

```diff
@@ -0,0 +1,53 @@
+(ns fluree.db.query.turtle.parse
```

Comment:
Definitely not for this pr, but I think we should eventually reorganize these namespaces. The `fluree.db.query.*` fragment is dealing with much more than queries.